runtime/binding-mqtt/src/test/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/client/ConnectionIT.java

```diff
@@ -34,7 +34,7 @@
 public class ConnectionIT
```

Comment:
These `ITs` will also need to move to ` .v5` subpackage, then `v4` subpackage will test mqtt 3.1.1 specification.